src/libraries/Common/src/Interop/Windows/SspiCli/SSPIWrapper.cs

```diff
@@ -141,6 +141,31 @@ public static SafeFreeCredentials AcquireCredentialsHandle(ISSPIInterface secMod
             return outCredential;
         }
 
+        public static unsafe SafeFreeCredentials AcquireCredentialsHandle(ISSPIInterface secModule, string package, Interop.SspiCli.CredentialUse intent, Interop.SspiCli.SCH_CREDENTIALS* scc)
+        {
+            if (NetEventSource.IsEnabled)
+            {
+                NetEventSource.Enter(null, package);
+                NetEventSource.Log.AcquireCredentialsHandle(package, intent, (IntPtr)scc);
+            }
+
+            SafeFreeCredentials? outCredential = null;
```

Comment:
Is the `= null` needed?

Reply:
no. I also updated legacy function I used as baseline.